tchannel/format.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import (
```

Comment:
I'm in favor of putting it inside the `formats` package. The `formats` package would basically expose standard formats supported by TChannel and the machinery to integrate your own custom formats.